DevProxy.Plugins/Reporting/GraphMinimalPermissionsGuidancePlugin.cs

```diff
@@ -33,7 +33,7 @@ public sealed class GraphMinimalPermissionsInfo
 
 public sealed class GraphMinimalPermissionsGuidancePluginConfiguration
 {
-    public IEnumerable<string>? PermissionsToExclude { get; set; }
+    public IEnumerable<string>? PermissionsToExclude { get; set; } = [];

```

Comment:
Let's not change this, because it changes the default behavior of the plugin.

Reply:
sure, let's leave initial value of PermissionsToExclude = null (as it was and as it is currently). 

>  if we set PermissionsToExclude to [] here, we won't exclude anything, but then the property is not set in the config which will cancel this and lead to reverting to default, right?

It is only correct if we change PermissionsToExclude property in _appsettings.json_.
In _GraphMinimalPermissionsGuidancePluginConfiguration_ class, either null or [] initial value does not change the behaviour.

This question bothered me with regards to the following feature where we are adding a similar permissions setting to MinimalPermissionsGuidancePlugin. I will implement it alike there.

Reply:
> This question bothered me with regards to the following feature where we are adding a similar permissions setting to MinimalPermissionsGuidancePlugin. I will implement it alike there.

Good catch. Let's do it in a separate PR